### ISSUE TITLE
Editor forgets Cel-copy sources on New Frame and New Empty Frame commands.

### ISSUE THREAD

**Issue by oceanhahn** [aseprite/aseprite#1865]
If a Cel or Cels are copied, and a new Frame is added before the Cels are pasted, the Editor clears the copy source. Not such a big problem on a small project, but when you need to copy many Cels at once or traverse a long Timeline, it can create a fair bit of extra nuisance. :s 

![copy-newframe-paste-error](https://user-images.githubusercontent.com/11544843/44948886-1a51e400-adf4-11e8-9a53-a9cdce025f84.gif)

Thank you for looking into it! :D 

..using ASEprite 1.9.2-x64 (installer) on Win10.

**Comment by kellis1990** [comment]
[github-recovery-codes.txt](https://github.com/aseprite/aseprite/files/2342668/github-recovery-codes.txt)

**Comment by dacap** [comment]
Yeah, actually the clipboard in the timeline (and the palette) is not handled as well as I would like (better feedback, using the real operating system clipboard so we can copy and paste between different Aseprite instances, etc.)
